tinygrad/opsgpu.py

```diff
@@ -377,27 +380,31 @@ def backward(ctx, grad_output):
 
 class AvgPool2D(Function):
   @staticmethod
-  def forward(ctx, input, kernel_size=(2, 2)):
+  def forward(ctx, input, kernel_size=(2, 2), stride=None):
```

Comment:
stride and kernel_size become ctx.stride and ctx.kernel_size by default

Reply:
Referencing as ctx.kernel_size in the backward pass now.